guess/touchscreen/action

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash -eu
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+. guess-functions
+
+$TRACE_SOURCE "Processing ..."
+
+# The idea comes from tslib. Touchscreen devices have
+# the INPUT_PROP_DIRECT property, which is determined
+# by the first from zero bit of the property number
+
+INPUT_PROP_DIRECT=1
+bitmask=$((1 << $INPUT_PROP_DIRECT))
+
+for i in "$SYSFS_PATH"/class/input/event*/device/properties; do
+	readline prop "$i"
+
+	if (($prop & $bitmask)); then
```

Comment:
Also it'd be better this way:
```
        readline prop "$i"
        (($prop & $bitmask)) || continue
        p="$(readlink -e "$i")"

        while [ "$p" != "$SYSFS_PATH"/devices ]; do
...
```

Everything else looks OK to me.

Reply:
Why? I don't see any pros except saving one line of code, but I think that trick with short circuit evaluation is bad, because it confuses not experienced people.

Reply:
The condition `(($prop & $bitmask))` covers all further logic of device processing. Enclosing this logic in an 'if' seems redundant. As @wladmis said, it seems more readable to me to first do a check and if it fails, do `continue`.

Reply:
That makes sense as a special case processing and for unify code style